dashboard/app/models/channel_token.rb

```diff
@@ -87,7 +87,7 @@ def self.find_channel_token(level, user_storage_id, script_id)
   def self.create_channel(ip, project, data: {}, src: nil, type: nil, remix_parent_id: nil, standalone: true, level: nil)
     if src
       data = project.get(src)
-      data.merge!(name: "Remix: #{data['name']}", hidden: false, frozen: false)
+      data.merge!(name: "Remix: #{data['name']}", hidden: false, frozen: false, publishedAt: nil)
```

Comment:
Out of curiosity, what happens if we don't include this? Does `publishedAt` default to a different value?

Reply:
When a project is remixed (`src` is defined), then when the new project is created, the merged data is assigned to the new project's `value` field. `published_at` IS assigned to `nil`. However, there is a copy of `published_at` stored in `value` which is out of sync, unless we reassign here.
The `project_at` in `value` is used in `projects_list.rb`.

Reply:
Related PR at https://github.com/code-dot-org/code-dot-org/pull/59781

Reply:
Yes, I meant `published_at` - fixed above.

After digging into this a little more, I found https://github.com/code-dot-org/code-dot-org/pull/16593 - `publishedAt` in sources was stored in redux and accessed when users were able to self-publish/unpublish. So this can be removed since users are no longer able self-publish ,and a published project now means that a user gives permission for their project to be displayed in a public gallery (moving from private -> public status).
I added removal as part of https://codedotorg.atlassian.net/browse/LABS-1185.
In `projects_list`, we assign in `get_project_row_data` `publishedAt` from `project[:published_at]` as expected (and not from `projectAt` strored in `value`).

Thanks to you both for this question!

Reply:
Thanks for clarifying! Tangential question (feel free to continue in Slack) - does this mean we have two `published_at` values, one as a field on the model, and one within the freetext `value` field?

Reply:
One last share, `publishedAt` in sources is assigned in legacy labs via `setPublishedAt` defined at https://github.com/code-dot-org/code-dot-org/blob/68b27d9ae228d06059856a984a03d00ecb9c9370/apps/src/code-studio/initApp/project.js#L1840.
It is called on in `header.js`.
I'll revert this change for now since it does not impact lab2 projects.